src/actions/controller.ts

```diff
@@ -8,7 +8,10 @@ export class ActionsController {
     public static loadActions() {
       vscode.extensions.all.forEach((extension) => {
         if (extension?.packageJSON?.BASContributes?.actions) {
-          this.actions.push(extension.packageJSON.BASContributes.actions);
+          const newActions: IAction[] = extension?.packageJSON?.BASContributes?.actions;
+          newActions.forEach((action) => {
+            this.actions.push(action);
+          });
```

Comment:
extension.packageJSON.BASContributes.actions.forEach((action) => {
            this.actions.push(action);
          });

Reply:
it needs to know that this is IAction[] type, it doesn't accept it without a declaration

Reply:
> it needs to know that this is IAction[] type, it doesn't accept it without a declaration

(extension.packageJSON.BASContributes.actions as IAction[]).forEach